projects/burnr/src/components/HistoryTable.tsx

```diff
@@ -138,9 +142,9 @@ const HistoryTable: React.FunctionComponent = () => {
 												{ // This may look overwhelming but is just for "dump" data until page is fixed
 												column.id === 'value'
 													&& typeof value === 'number'
-													&& <BalanceValue value={
-														new TypeRegistry().createType('Balance', new BN(value))
-													} />}
+													&& <BalanceValue
+														value={new TypeRegistry().createType('Balance', new BN(value))}
```

Comment:
The Type `Balance` is already registered and known by the Polkadot JS API, you only need to register new, unknown types. You can just pass the `value` here as it is already defined as Type `Balance` in the `BalanceValue` component. https://github.com/paritytech/substrate-connect/blob/3d5eb9be27b18a920a391620e74f894d22a25880/projects/burnr/src/components/BalanceValue.tsx#L10

If you want to use the Type, you can just import the type from the API:
```
import { Balance } from '@polkadot/types/interfaces';
```